project_charter/views/project_charter_view.xml

```diff
@@ -95,7 +95,6 @@
                                                 type="object"
                                                 icon="gtk-edit"/>
                                         <field name="complete_wbs_code" string="Code"/>
-                                        <field name="wbs_indent"/>
```

Comment:
Why remove indent field? It's handy to be able to visualize the WBS level of the element.